<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Andy Styles</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1644</width>
    <height>1037</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="terminalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1644</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Session</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionReconnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionSessionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuFont_Settings">
    <property name="title">
     <string>Themes</string>
    </property>
    <addaction name="actionKeyboardThemes"/>
    <addaction name="actionColourThemes"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentSessions">
     <property name="title">
      <string>Recent Sessions</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_Session"/>
    <addaction name="actionOpen_Duplicate_Session"/>
    <addaction name="actionSave_Session"/>
    <addaction name="actionSave_SessionAs"/>
    <addaction name="menuRecentSessions"/>
    <addaction name="separator"/>
    <addaction name="actionManage_Sessions"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTabs"/>
    <addaction name="actionTile"/>
    <addaction name="actionGrid"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Q3270"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnect"/>
   <addaction name="menuFont_Settings"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionConnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Connect...</string>
   </property>
  </action>
  <action name="actionSessionPreferences">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Window...</string>
   </property>
  </action>
  <action name="actionTabs">
   <property name="text">
    <string>Tabs</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Q3270">
   <property name="text">
    <string>About Q3270...</string>
   </property>
  </action>
  <action name="actionKeyboardThemes">
   <property name="text">
    <string>Keyboard</string>
   </property>
  </action>
  <action name="actionColourThemes">
   <property name="text">
    <string>Colours</string>
   </property>
  </action>
  <action name="actionSave_SessionAs">
   <property name="text">
    <string>Save Session As...</string>
   </property>
  </action>
  <action name="actionOpen_Session">
   <property name="text">
    <string>Open Session...</string>
   </property>
  </action>
  <action name="actionOpen_Duplicate_Session">
   <property name="text">
    <string>Open Duplicate Session</string>
   </property>
  </action>
  <action name="actionManage_Sessions">
   <property name="text">
    <string>Manage Sessions...</string>
   </property>
  </action>
  <action name="actionSave_Session">
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionSessionPreferences</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuSessionPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>822</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>822</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>822</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDisconnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuDisconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReconnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuReconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_Q3270</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionColourThemes</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuColourTheme()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_SessionAs</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuSaveSessionAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Session</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuOpenSession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionKeyboardThemes</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuKeyboardTheme()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Duplicate_Session</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuDuplicate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionManage_Sessions</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuManageSessions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Session</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuSaveSession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menuConnect()</slot>
  <slot>menuQuit()</slot>
  <slot>menuSessionPreferences()</slot>
  <slot>menuSetFont()</slot>
  <slot>menuDisconnect()</slot>
  <slot>menuNew()</slot>
  <slot>menuTabbedView(bool)</slot>
  <slot>menuReconnect()</slot>
  <slot>menuAbout()</slot>
  <slot>menuColourTheme()</slot>
  <slot>menuSaveSessionAs()</slot>
  <slot>menuOpenSession()</slot>
  <slot>menuKeyboardTheme()</slot>
  <slot>menuDuplicate()</slot>
  <slot>menuManageSessions()</slot>
  <slot>menuSaveSession()</slot>
 </slots>
</ui>
